clip_retrieval/clip_inference/distributor.py

```diff
@@ -35,14 +44,19 @@ def __call__(self):
                 .getOrCreate()
             )
 
-        df = list(range(self.output_partition_count))
-        rdd = spark.sparkContext.parallelize(df, len(df))
+        df = list(range(self.worker_args["output_partition_count"]))
+        # NOTE: https://spark.apache.org/docs/1.0.1/programming-guide.html
```

Comment:
Please remove that

Reply:
217398fda4be205d4afe6fbac8c76f055fa04e05